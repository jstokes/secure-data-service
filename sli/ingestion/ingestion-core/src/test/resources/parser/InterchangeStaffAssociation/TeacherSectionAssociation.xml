<?xml version="1.0" encoding="UTF-8"?>
<InterchangeStaffAssociation xmlns:sli="http://slc-sli/ed-org/0.1" xmlns="http://ed-fi.org/0100"
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <TeacherSectionAssociation>
    <TeacherReference>
      <StaffIdentity>
        <StaffUniqueStateId>pXas4CbbfwQQi</StaffUniqueStateId>
      </StaffIdentity>
    </TeacherReference>
    <SectionReference>
      <SectionIdentity>
        <EducationalOrgReference>
          <EducationalOrgIdentity>
            <StateOrganizationId>A2GmaqA5BhrhWmBw6J_5ow</StateOrganizationId>
          </EducationalOrgIdentity>
        </EducationalOrgReference>
        <UniqueSectionCode>bjoYzn6NwaVpdHYLbRpGkaxkkIYDHMfC6XAfLTAZZr-Q</UniqueSectionCode>
      </SectionIdentity>
    </SectionReference>
    <ClassroomPosition>Substitute Teacher</ClassroomPosition>
    <BeginDate>2014-03-29</BeginDate>
    <EndDate>2013-11-14</EndDate>
    <HighlyQualifiedTeacher>true</HighlyQualifiedTeacher>
  </TeacherSectionAssociation>
</InterchangeStaffAssociation>
