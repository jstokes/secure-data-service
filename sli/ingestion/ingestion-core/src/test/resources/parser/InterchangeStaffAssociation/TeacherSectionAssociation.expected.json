{
  "TeacherReference":{
    "StaffIdentity":{
        "StaffUniqueStateId":{"_value":"pXas4CbbfwQQi"}
      }
  },
  "SectionReference":{
    "SectionIdentity":{
      "EducationalOrgReference":{
        "EducationalOrgIdentity":{
          "StateOrganizationId":{"_value":"A2GmaqA5BhrhWmBw6J_5ow"}
        }
      },
      "UniqueSectionCode":{"_value":"bjoYzn6NwaVpdHYLbRpGkaxkkIYDHMfC6XAfLTAZZr-Q"}
    }
  },
  "ClassroomPosition":{"_value":"Substitute Teacher"},
  "BeginDate":{"_value":"2014-03-29"},
  "EndDate":{"_value":"2013-11-14"},
  "HighlyQualifiedTeacher":true
}